polaris-react/src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -315,10 +320,7 @@ export class PositionedOverlay extends PureComponent<
           preferredAlignment,
         );
 
-        const chevronOffset =
-          activatorRect.center.x -
-          horizontalPosition +
-          overlayMargins.horizontal * 2;
```

Comment:
I think we need to keep this `overlayMargins.horizontal * 2` 👍 this was part of a recent tooltip fix

Reply:
Ah, sorry thank you for the clarification! Sounds good 👍